src/isomorphic/React.js

```diff
@@ -62,6 +63,10 @@ var React = {
     only: onlyChild,
   },
 
+  CSS: {
+    multi: CSSPropertyOperations.multi,
+  },
```

Comment:
I'd actually even argue that this shouldn't be a part of React's API and should live in a little helper/addon module somewhere on NPM.